plugins/API/ProcessedReport.php

```diff
@@ -715,9 +715,17 @@ private function aggregateReportTotalValues($simpleDataTable, $totals)
         $simpleTotals = $this->hideShowMetrics($metadataTotals);
 
         foreach ($simpleTotals as $metric => $value) {
+            if (0 === strpos($metric, 'avg_') || '_rate' === substr($metric, -5) ||
+                0 === strpos($metric, 'min_') || 0 === strpos($metric, 'max_')) {
+                continue; // skip average, minimum, maximum and rate metrics
+            }
+            if (is_array($value) ) {
+                continue; // skip totals for nested metrics
+            }
+
             if (!array_key_exists($metric, $totals)) {
                 $totals[$metric] = $value;
-            } else {
+            } else if(is_numeric($value)) {
```

Comment:
did you see why there are non numeric values? maybe we need to fix something? Just wanting to avoid we just "hide" the actual problem

Reply:
the failures I've seen seems to be because of percentage values e.g. `15%` or nested data in arrays. So with the checks above it wouldn't be needed anymore. But as it could still for custom plugins we maybe should do the check nevertheless